contributing_to_docs/doc_guidelines.adoc

```diff
@@ -743,21 +743,84 @@ Replace all public IP addresses with an address from the following blocks. These
 
 [IMPORTANT]
 ====
-Do not split link paths across lines when wrapping text. This will cause issues with the doc builds.
+Do not split link paths across lines when wrapping text. This will cause issues with the documentation builds.
 ====
 
+[id="lead-in-link-sentences"]
+=== Lead-in link sentences
+
+Using a lead-in link sentence that adheres to guidelines maintains consistency across OpenShift Container Platform documentation.
+
+==== Lead-in link sentence style for non-xref situations
+
+For situations where you cannot use the `xref` attribute, such as referencing a module within a module, ensure that you adhere to the following guidance for lead-in link sentences.
+
+* Use the following format when pointing to a section in the same document:
++
+[source,text,subs="+quotes"]
+----
+For more information, see "<section_name>".
+----
++
+* Wrap the `<section_name>` string in quotation marks. The section name can be the title of a module or an assembly.
+* Do not state "in the Additional Resources section" or other variants. Mentioning the `<section_name>` is enough to prompt a documentation user to find the section in the same document.
+
+==== Lead-in link sentence style for the xref attribute
+
+For situations where you can use the `xref` attribute, adhere to the following guidance:
+
+* When pointing to a section that exists in the same documentation set, create an _Additional resources_ section in the assembly file and then place the `xref` as a bulleted item under this section. For example:
++
+[source,text,subs="+quotes"]
+----
+[role="_additional-resources"]
+.Additional resources
+* xref:../../../installing/installing_vsphere/installation-config-parameters-vsphere.adoc#installation-config-parameters-vsphere[Installation configuration parameters]
+----
+* When pointing to a link in a section that exists in another OpenShift Container Platform document, use the following format:
++
+[source,text,subs="+quotes"]
+----
+For more information on <topic>, see xref:["<section_name>"] in _<document_name>_.
+----
++
+Note the quotation marks around `<section_name>` and the italics around `<document_name>`.
+
+For more information about using `xref`, see xref:links-conditionals-assemblies[Links, hyperlinks, and cross references].
+
+==== Lead-in link sentence style for the link attribute
+
+For situations where you can use the `link` attribute, adhere to the following guidance:
+
+* When pointing to a link that exists external to Red Hat's customer portal documentation, use the `link` attribute, place the link in an _Additional resources_ section, and use the `<document_heading_name> (<document_source>)` format. For example:
++
+[source,text,subs="+quotes"]
+----
+See link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features] (AWS documentation).
+----
```

Comment:
So this block is _not_ showing the _Additional resources_ part of the example, and is instead showing "use the `<document_heading_name> (<document_source>)` format" from within the running body text as well? Apologies if I'm not connecting the dots correctly.

Reply:
I did not want to call out that one can create a module-level or assembly-level Add. resources section so I did not include an example in the code block. Should I?

Reply:
@adellape

Reply:
> I did not want to call out that one can create a module-level or assembly-level Add. resources section

Is the operative word there "create"? Since it does mention "place the link in an _Additional resources_ section". Sorry, feel like just I'm splitting hairs at this point, I don't think we need to drag it out any longer. 🙃 

> so I did not include an example in the code block. Should I?

I don't think so. I think "place the link in an _Additional resources_ section" is clear enough what they should do, just wanted to make sure I understood the full set of things we're saying to do.

Reply:
OK actually, I have now just read the later thread here https://github.com/openshift/openshift-docs/pull/73514#pullrequestreview-2001169258

Reply:
@adellape . Is the PR right for merging; need another review; item pending?